pandas/tests/series/test_constructors.py

```diff
@@ -1347,12 +1347,11 @@ def test_convert_non_ns(self):
     def test_constructor_cant_cast_datetimelike(self, index):
 
         # floats are not ok
-        msg = "Cannot cast {}.*? to ".format(
-            # strip Index to convert PeriodIndex -> Period
-            # We don't care whether the error message says
-            # PeriodIndex or PeriodArray
-            type(index).__name__.rstrip("Index")
-        )
+        msg = f"Cannot cast {type(index).__name__.rstrip('Index')}.*? to "
+        # strip Index to convert PeriodIndex -> Period
```

Comment:
I guess should move this comment before 1350 so it reads sequentially

Reply:
Thanks, I was wondering about this, too